.github/workflows/docker.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker unstable release
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  ssr:
+    name: Build SSR docker image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Configure QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Configure Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: |
+            jellyfin/jellyfin-vue:unstable-ssr
+            jellyfin/jellyfin-vue:unstable-ssr.${{ steps.date.outputs.date }}.${{ github.sha }}
```

Comment:
Is there a reason for tagging with the date?

Reply:
I thought that it gave more freedom when looking for images in docker hub, so one doesn't need to go back all the git history to check for an specific day. We have dates right now in our pushed tags, so I think it makes sense to keep them somehow.

With date and short hash, I think everyone will be pleased :D

Reply:
FYI: `github.sha` is the full hash not the short hash

Reply:
Maybe we can put the short one then @ferferga ? :p

Reply:
the only way to get a short SHA would be to build it yourself in a step before use. [Something I stumbled over](https://stackoverflow.com/a/59819441) while I searched for this functionality for another repo. 

(note: I ended up not using the SHA so not 100% sure It would work, but looked good to me)

Reply:
@h1dden-da3m0n Done!